### PR TITLE
Fix library add ios with spaces in path

### PR DESCRIPTION
**Pull request by jasssonpet**
Fixes https://github.com/NativeScript/nativescript-cli/issues/459 in the CLI

### CODE REVIEW

File: lib/services/ios-project-service.ts [hunk 1]
```diff
@@ -230,7 +230,7 @@ class IOSProjectService implements  IPlatformProjectService {
                 this.$errors.failWithoutHelp("The bundle at %s does not contain an Info.plist file.", libraryPath);
             }
 
-            var packageType = this.$childProcess.exec('/usr/libexec/PlistBuddy -c "Print :CFBundlePackageType" ' + infoPlistPath).wait().trim();
+            var packageType = this.$childProcess.exec(util.format('/usr/libexec/PlistBuddy -c "Print :CFBundlePackageType" "%s"', infoPlistPath)).wait().trim();
```
Comment (ligaz) [inline lib/services/ios-project-service.ts]: You can use string interpolation here. It makes the code more readable.

Comment (jasssonpet) [inline lib/services/ios-project-service.ts]: Nice :)

**Comment by teobugslayer** [comment]
For this to work, you need to use back quotes (`)

**Comment by Fatme** [comment]
:+1:

**Pull request merged**